src/net.cpp

```diff
@@ -962,9 +974,15 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     uint64_t nonce = GetDeterministicRandomizer(RANDOMIZER_ID_LOCALHOSTNONCE).Write(id).Finalize();
     CAddress addr_bind = GetBindAddress(hSocket);
 
-    CNode* pnode = new CNode(id, nLocalServices, GetBestHeight(), hSocket, addr, CalculateKeyedNetGroup(addr), nonce, addr_bind, "", true);
+    ServiceFlags nodeServices = nLocalServices;
+    if (NetPermissions::HasFlag(permissionFlags, PF_BLOOMFILTER)) {
+        nodeServices = static_cast<ServiceFlags>(nodeServices | NODE_BLOOM);
```

Comment:
Where is this from and why is it needed? I couldn't find it in previous code

Reply:
The initial problem I wanted to solve is that, because in 0.19 we remove BLOOM_FILTER by default, I wanted the ability to keep it only for whitelisted peer.  Because this was a simple change, I decided to add it here instead of separate PR.

Reply:
Ah, I figured. Was a bit weird to see this behavior change in a "refactoring" commit